src/core/image.js

```diff
@@ -332,10 +323,10 @@ class PDFImage {
       // form, so we can just transfer it.
       data = imgArray;
     } else if (!inverseDecode) {
-      data = new Uint8ClampedArray(actualLength);
+      data = new Uint8Array(actualLength);
       data.set(imgArray);
     } else {
-      data = new Uint8ClampedArray(computedLength);
+      data = new Uint8Array(computedLength);
       data.set(imgArray);
       for (i = actualLength; i < computedLength; i++) {
```

Comment:
nit: use `data.fill` even if it's unrelated to your patch. But it can later (or never) if you prefer.

Reply:
Well, that cannot hurt to change.
(This code most likely pre-dates the introduction of `TypedArray.prototype.fill()` in the standard.)